src/deepsparse/image_classification/schemas.py

```diff
@@ -38,10 +38,15 @@ class ImageClassificationOutput(BaseModel):
     """
     Output model for image classification
     """
+    ndarray: numpy.ndarray = numpy.ndarray
+
+    class Config:
+        arbitrary_types_allowed = True
 
     labels: List[Union[int, str, List[int], List[str]]] = Field(
         description="List of labels, one for each prediction"
     )
     scores: List[Union[float, List[float]]] = Field(
         description="List of scores, one for each prediction"
     )
+    logits: Optional[ndarray] = Field(description="Raw model outputs")
```

Comment:
Could we test this logic with deepsparse server? I remember we had issues earlier when keeping numpy arrays in I/O schemas

Reply:
Yes, I second that!

Reply:
also good idea to set the default to None here